runner/src/main.rs

```diff
@@ -361,12 +359,10 @@ fn build_server(opt: &BuildServer) -> Step {
                             "--out-dir=oak_loader/bin".to_string(),
                             // `--out-dir` is unstable and requires `-Zunstable-options`.
                             "-Zunstable-options".to_string(),
-                            ...if opt.server_variant == "logless" {
-                                vec!["--no-default-features".to_string()]
-                            } else if opt.server_variant == "no-introspection-server" {
-                                vec![]
-                            } else {
-                                vec!["--features=oak_introspection_client".to_string()]
+                            ...match opt.server_variant {
+                                ServerVariant::Logless => vec!["--no-default-features".to_string()],
+                                ServerVariant::NoIntrospectionServer => vec![],
+                                ServerVariant::Base => vec!["--features=oak_introspection_client".to_string()],
```

Comment:
That's nice. :) Was about to suggest this while reviewing the previous commit. :p

Reply:
Yeah, I should have done the commits the other way around but I don't think it's worth unwinding now.

Reply:
(I've squashed the two commits together)